pkg/cmd/pr/merge/http.go

```diff
@@ -100,6 +100,43 @@ func disableAutoMerge(client *http.Client, repo ghrepo.Interface, prID string) e
 	return gql.MutateNamed(context.Background(), "PullRequestAutoMergeDisable", &mutation, variables)
 }
 
+func getMergeHeadlineText(client *http.Client, repo ghrepo.Interface, prID string, mergeMethod PullRequestMergeMethod) (string, error) {
+	var method githubv4.PullRequestMergeMethod
+	switch mergeMethod {
+	case PullRequestMergeMethodMerge:
+		method = githubv4.PullRequestMergeMethodMerge
+	case PullRequestMergeMethodRebase:
+		method = githubv4.PullRequestMergeMethodRebase
+	case PullRequestMergeMethodSquash:
+		method = githubv4.PullRequestMergeMethodSquash
+	}
+
+	var query struct {
+		Node struct {
+			PullRequest struct {
+				ViewerMergeHeadlineText string `graphql:"viewerMergeHeadlineText(mergeType: $method)"`
+			} `graphql:"...on PullRequest"`
+		} `graphql:"node(id: $prID)"`
+	}
+
+	variables := map[string]interface{}{
+		"prID":   githubv4.ID(prID),
+		"method": method,
+	}
+
+	gql := graphql.NewClient(ghinstance.GraphQLEndpoint(repo.RepoHost()), client)
+	err := gql.QueryNamed(context.Background(), "PullRequestMergeHeadlineText", &query, variables)
+	if err != nil {
+		// Tolerate this API missing in older GitHub Enterprise
+		if strings.Contains(err.Error(), "Field 'viewerMergeHeadlineText' doesn't exist") {
+			return "", nil
```

Comment:
Same as `viewerMergeBodyText`, the `viewerMergeHeadlineText` might not be available in older GHE versions.